test/e2e/config/docker.yaml

```diff
@@ -5,17 +5,17 @@
 # - control-plane kubeadm
 # - docker
 
-# For creating local dev images run ./scripts/ci-e2e.sh
+# For creating local dev images run make docker-build-e2e from the main CAPI repository
```

Comment:
I think we can keep `run ./scripts/ci-e2e.sh` here, because running `make docker-build-e2e` will not create images with the tag `dev` this file is using.

Reply:
moved to #3651